src/app/@ansyn/ansyn/package.json

```diff
@@ -2,39 +2,41 @@
 	"name": "@ansyn/ansyn",
 	"version": "3.0.0",
 	"peerDependencies": {
-		"@ngrx/core": "^1.2.0",
-		"@ngrx/store": "^9.0.0",
-		"@ngrx/effects": "^9.0.0",
-		"@angular/common": "~10.0.5",
-		"@angular/core": "~10.0.5",
+		"@ngrx/store": "9.x",
+		"@ngrx/effects": "9.x",
+		"@angular/common": "10.0.0-PLACEHOLDER",
```

Comment:
what does placeholder mean?

Reply:
it should say to the customer to install any angular version he want but it should be equal to all.
I change it to 9 or 10.